docs/docs/dev_docs/privacy/main.md

```diff
@@ -0,0 +1,177 @@
+# Preserving privacy
+
+Privacy is important.
+
+Keeping information private is difficult.
+
+Once information is leaked, it cannot be unleaked.
+
+---
+
+If you navigated to this page via a search engine, or by social media, those sites probably collected some data about that action you took. Your internet service provider (ISP) knows you're reading this page, unless you're using a VPN. The VPN provider will know you're reading this page, unless you're using Tor. Your entry and exit nodes for Tor will know you're reading this page, but might not know who you are. That is, unless you navigated here from social media!
+
+Have you ever tried to use the internet without leaking any information about yourself?
+
+Take a moment to imagine how you might purchase a subscription to an online newspaper, without revealing anything about yourself or your activity to anyone. Not to your bank, to visa, your ISP, your VPN, the newspaper company, the search engine.
+
+The point? To keep information private requires careful consideration.
+
+A more exact point: to write a private smart contract requires careful consideration.
+
+---
```

Comment:
Might have got carried away here. Upon reflection, maybe I should delete this rambling section?